adapters/gothamads/gothamads_test.go

```diff
@@ -0,0 +1,28 @@
+package gothamads
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestJsonSamples(t *testing.T) {
+	bidder, buildErr := Builder(openrtb_ext.BidderGothamads, config.Adapter{
+		Endpoint: "http://us.example.com/?pass={{.AccountID}}"}, config.Server{ExternalUrl: "http://hosturl.com", GvlID: 1, DataCenter: "2"})
```

Comment:
just as a practise. lets match endpoint with `http://us-e-node1.gothamads.com/?pass={{.AccountID}}` mentioned in   bidder-info yaml

Reply:
fixed